cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/AbstractQueueAdmin.java

```diff
@@ -79,43 +60,32 @@ public static String getFlowName(String queueTableName) {
     return parts[parts.length - 1];
   }
 
+  public TableId getConfigTableId(QueueName queueName) {
+    return getConfigTableId(queueName.getFirstComponent());
+  }
+
+  public TableId getConfigTableId(String namespace) {
+    return TableId.from(root, namespace, unqualifiedConfigTableNameSuffix);
```

Comment:
Also, since its not on the same topic (And to keep this PR small and merged without taking too long), I prefer to do this in a separate PR.